tests/routers/test_auth.py

```
@@ -6,6 +6,9 @@
 from app.models import User
 from app.security import hash_password
 
+# Use a constant for password to avoid Security Hotspot
+TEST_PASSWORD = "password"  # NOSONAR
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "TEST_PASSWORD"

(S105)